applications/vessel-history/src/types/activity.ts

```diff
@@ -55,7 +55,9 @@ export interface ActivityEvent {
 }
 export interface ActivityEventGroup {
     event_type: EventType
-    event_place?: string
+    event_eez?: number | string
+    event_rfmo?: number | string
+    ocean?: string
```

Comment:
I'd try to do this more generic to include any region and not having to explicitly modify this type  in the future.

Reply:
I can add an array but will add lot logic to make build the groups, and render data. maybe make sense, I will take a look